data/korath ships.txt

```diff
@@ -955,6 +968,23 @@ ship "Model 16"
 	description "The Kor Mereti war drones are constructed in a modular fashion, which allows them to grow over time as more materials become available. This one is recently built, and relatively small."
 
 
+ship "Model 16" "Model 16 (Jump Miner)"
```

Comment:
And as implied by other stuff  but not officially stated near the definition, this variant and the jump drive model 8 are only used by the Korath Exiles after they steal the drone tech. Since they have jump drive capability, these and the Sestor drones can too.